ViewModel/AggregatedStageDamage/AggregatedStageDamageElement.cs

```diff
@@ -16,8 +16,9 @@ public class AggregatedStageDamageElement : ChildElement
     {
         #region Properties
```

Comment:
Yes, we talked about it, but I didn't change any code. I guess I don't really see the benefit. Let's think through an example. We have a "Student" that has a Name and GPA property. The StudentEditor.xaml needs to bind to the Name and GPA property. The StudentEditorVM needs to have a Name and GPA property. How would you like to see the save work?

I think you are proposing (based on what you did for graphical) that the "Student" object would hold the "StudentEditorVM" object. I am generally free any time if you want to talk about this more. I think it might help to have someone more experienced with architecture like Will or Ryan take a look. I am basically following the pattern that was set down years ago for fda. It might be pretty time consuming but since i am refactoring all the save logic, it might be a good time to think about this.

Reply:
The way fda works now is that the StudentEditorVM creates a new Student element and then saves that to the database, which notifies the cache, which then swaps out the Student with that ID. We are never really updating any of the elements, we just swap them out.